src/app-exception-filters/main-exception.filter.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  ExceptionFilter,
+  Catch,
+  ArgumentsHost,
+  HttpException,
+  Logger,
+  LoggerService,
+} from '@nestjs/common';
+import { Response } from 'express';
+
+@Catch()
+export class MainExceptionFilter implements ExceptionFilter {
+  private readonly logger: LoggerService = new Logger(MainExceptionFilter.name);
+
+  catch(exception: Error, host: ArgumentsHost) {
+    const ctx = host.switchToHttp();
+    const response = ctx.getResponse<Response>();
+    const status =
+      exception instanceof HttpException ? exception.getStatus() : 500;
+
+    this.logger.error(exception.stack);
```

Comment:
dalej nie rozumiem czemu na sile wrzucacie tutaj stack - on jest brany by defauly przy logowaniu - przekazcie caly blad po prostu

Reply:
dam wam juz approve ale poprawcie to